src/current/v23.2/cluster-virtualization-overview.md

```diff
@@ -0,0 +1,104 @@
+- When a cluster setting is scoped to the system interface, it effects the entire storage cluster. To configure a cluster setting that is scoped to the system interface, you must have the `admin` role on the system interface, and you must connect to the system interface before configuring the setting.
+- When a cluster setting is system-visible, it can be set only from the system interface but can be queried from any virtual cluster. For example, virtual cluster can query a system-visible cluster setting's value to help adapt to the storage cluster's configuration.
+
+For more details, including the scope of each cluster setting, refer to [Cluster Setting Scopes with Cluster Virtualization enabled]({% link {{ page.version.version }}/cluster-virtualization-setting-scopes.md %}).
+
+### Security
+
+By default, when connecting to a cluster using a SQL client or the DB Console, if a user does not specify whether to connect to a virtual cluster or the system interface, they are connected to the virtual cluster.
+
+To grant access to the system interface, you must connect to the system interface as a user with the `admin` role, then grant the `admin` role to the SQL user. For details, refer to [Work with virtual clusters]({% link {{ page.version.version }}/work-with-virtual-clusters.md %}#grant-access-to-the-system-interface).
+
+### Upgrades
+
+When cluster virtualization is enabled to upgrade to a new major version, you must first replace the binary on each node and restart the node, [finalize] the upgrade on the system interface to upgrade it, then finalize the upgrade on the virtual cluster to upgrade it. This allows you to roll back an upgrade of the system interface without impacting schemas or data in virtual clusters. The system interface can be at most one major version ahead of virtual clusters. For example, when v24.1 is released, a system interface on CockroachDB v24.1 can have virtual clusters on CockroachDB v23.2.
+
+For details, refer to [Work with virtual clusters]({% link {{ page.version.version }}/work-with-virtual-clusters.md %}#upgrade-a-cluster).
+
+### Disaster recovery
+
+When cluster virtualization is enabled, the default scope of [backup]({% link {{ page.version.version }}/backup.md %}) and [restore]({% link {{ page.version.version }}/restore.md %}) commands is the virtual cluster. This means that:
+
+- By default, backups taken from a virtual cluster contain only data for that virtual cluster, not the system interface.
+- To back up the entire cluster, you must connect to the system interface to run the backup and include the `INCLUDE_ALL_SECONDARY_TENANTS` flag.
```

Comment:
This flag is unfortunately named.  @stevendanna do we have an issue open to rename this to be VC-based?

Reply:
`include_all_virtual_clusters` should also work.

However, I think we want slightly different advice that doesn't focus on this flag.

Something like:

- A backup taken from a virtual cluster contains all data for that virtual cluster, but does not contain modifications made via the system interface such as system-level cluster settings.
- If your deployment contains system-level customisations, a separate backup may be taken from the system interface to capture those customisations.

The `include_all_virtual_clusters` option is intended primarily for CC Serverless use and is subject to change.